pkg/amphoracontrollers/flavors.go

```diff
@@ -230,7 +239,12 @@ func ensureFlavors(osclient *openstack.OpenStack, log *logr.Logger, instance *oc
 			log.Info(fmt.Sprintf("Creating Octavia flavor profile \"%s\"", flavorProfileCreateOpts.Name))
 			fp, err := flavorprofiles.Create(lbClient, flavorProfileCreateOpts).Extract()
 			if err != nil {
-				return "", err
+				errFmt := fmt.Errorf("error creating flavor profile: %w", err)
+				log.Error(errFmt, fmt.Sprintf("Amphora image might be missing or not "+
```

Comment:
Maybe a log.Warning would be more appropriate

Reply:
log.Logger does not seem to have a warning level. I change this to Info. https://pkg.go.dev/github.com/go-logr/logr@v1.4.2#hdr-Usage